### ISSUE TITLE
Confused over variable overriding

### ISSUE THREAD

**Issue by rossburton** [priv-kweihmann/oelint-adv#204]
If I oelint the optee recipes:

http://git.yoctoproject.org/cgit/cgit.cgi/meta-arm/tree/meta-arm/recipes-security/optee

```
optee-examples.inc:42:error:oelint.var.override:Variable 'PACKAGE_ARCH' is set by optee-test.inc,optee-os.inc,optee-examples.inc
```

Yes.  Why is it telling me this? They are independent recipes.

**Comment by priv-kweihmann** [comment]
Basically it says that `PACKAGE_ARCH = "${MACHINE_ARCH}"` is set by all of the 3 mentioned file, which are all included in the same recipe - for me `PACKAGE_ARCH` might be uncritical, but this is a more general rule, which should catch those nifty overrides from inc-files.
But yeah I agree wording could be improved - any ideas?

**Comment by priv-kweihmann** [comment]
but let me check that in detail...

**Comment by priv-kweihmann** [comment]
Deng, I see and let me guess you did run the tool like `oelint-adv <options> *.bb`, right?
In that case yes, this is a false positive

**Comment by rossburton** [comment]
Yes, I did *.bb.
